website/src/components/Edit/InputForm.tsx

```diff
@@ -259,6 +273,12 @@ export const SequencesForm: FC<SequenceFormProps> = ({ editableSequences, setEdi
                             ariaLabel={`${field.key} Segment File`}
                             fileKind={PLAIN_SEGMENT_KIND}
                             small={true}
+                            initialValue={
+                                field.initialValue.length > 0
+                                    ? new VirtualFile(field.initialValue, 'existing_data.txt')
```

Comment:
```suggestion
                                    ? new VirtualFile(field.initialValue, 'Existing data')
```
Would this still work? IMO it would look a bit better

Reply:
yes taht works! Done.